xbmc/platform/linux/ScreenshotSurfaceAML.cpp

```diff
@@ -51,31 +64,31 @@ void CScreenshotAML::CaptureVideoFrame(unsigned char *buffer, int iWidth, int iH
     {
       if (!bBlendToBuffer)
       {
-        memset(buffer, 0xff, buffSize);
+        memset(m_buffer, 0xff, buffSize);
       }
 
-      for (int y = 0; y < iHeight; ++y)
+      for (int y = 0; y < m_height; ++y)
       {
         unsigned char *videoPtr = videoBuffer + y * stride;
 
-        for (int x = 0; x < iWidth; ++x, buffer += 4, videoPtr += 3)
+        for (int x = 0; x < m_width; ++x, m_buffer += 4, videoPtr += 3)
         {
-          float alpha = buffer[3] / (float)255;
+          float alpha = m_buffer[3] / (float)255;
 
           if (bBlendToBuffer)
           {
             //B
-            buffer[0] = alpha * (float)buffer[0] + (1 - alpha) * (float)videoPtr[0];
+            m_buffer[0] = alpha * (float)m_buffer[0] + (1 - alpha) * (float)videoPtr[0];
```

Comment:
normally i'd complain about c casts; in this case remove the casts altogether, alpha makes things float.